jenkins/cori_atmnbfb.sh

```diff
@@ -5,4 +5,11 @@ export SCRIPTROOT=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && cd .. && pwd )
 export CIME_MACHINE=cori-knl
 source $SCRIPTROOT/util/setup_common.sh
 
+# Unload python module from:
+# $SCRIPTROOT/util/setup_common.sh --> $SCRIPTROOT/util/${CIME_MACHINE}_setup.sh
+module unload python/2.7-anaconda-5.2
+
+# Get cime_env conda env
+source /global/project/projectdirs/acme/software/anaconda_envs/load_latest_cime_env.sh
+
```

Comment:
Do you have to change this file?   I thought it was used only for nbfb tests which don't have the problem that maint-1.0 has (since they are using next).

Reply:
Yes. nb4b tests need `cime_env` so to have cori default to something else (in `cori-knl_setup.sh`), it has to be loaded here.

Alternatively, I could only load the python 2 module for maint in `cori_prod_maint.sh`, but you indicated in this comment I should walk all of cori back: https://github.com/E3SM-Project/E3SM_test_scripts/pull/13#issuecomment-552574974

Reply:
I've been assuming cori-knl.setup.sh and cori-atmnbfb.sh are independent of each other.  Go ahead and just make an exception for maint-1.0 if thats possible.

Reply:
Yeah, it goes `setup_common.sh` -> `[MACHINE]_setup.sh` -> `[TEST_SCRIPT].sh`